apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -148,13 +153,17 @@ export default function Chat({
     let response = ''
     let messagesCount = 0
 
-    const { output } = await runDocumentAction({
+    const { response: actionResponse, output } = await runDocumentAction({
       projectId: project.id,
       documentPath: document.path,
       commitUuid: commit.uuid,
       parameters,
     })
 
+    actionResponse.then((res) => {
+      setDocumentLogUuid(res.response.documentLogUuid)
```

Comment:
When SDK (used in our server action) finishes it returns `documentLogUuid` which we set in the state and use when the user keeps talking in the chat.